### ISSUE TITLE
Proposition to remove the healing axe as quest reward

### ISSUE THREAD

**Issue by jackowski626** [GTNewHorizons/GT-New-Horizons-Modpack#7097]
* First: it is **OP**
* Thanks to @mitchej123 we are getting an aid in crafting various foods
* With the axe, making food becomes **completely useless** when you gather some gardens. There is no point in keeping Pam's
* The player should be **incentivized** to automate the creation of various foods

![image](https://user-images.githubusercontent.com/22838328/102010603-5b5d3080-3d3f-11eb-8d2b-a27ea476f6a2.png)

**Comment by KiloJoel** [comment]
Eating a variety of food is still incentivised by gaining hearts from eating unique foods.

**Comment by jackowski626** [comment]
yes, crafting a lot once and running while jumping to eat them the fastest one can :concern:

**Comment by repo-alt** [comment]
unbreakable axe is fine btw, to use in EIO tree farm, just disable the healing part

**Comment by KiloJoel** [comment]
> 
> 
> unbreakable axe is fine btw, to use in EIO tree farm, just disable the healing part

I am very much against disabling the hunger regen. This is a nice QOL thing for the full game. Not everyone wants to still be worrying about having food in their inventory all the time at high tiers.

**Comment by repo-alt** [comment]
idk, I usually use autocrafted tin cans lategame

**Comment by jackowski626** [comment]
this

**Comment by KiloJoel** [comment]
options are important, don't make only 1 reasonable solution by removing other ones

**Comment by boubou19** [comment]
> Eating a variety of food is still incentivised by gaining hearts from eating unique foods.

Well a fast way to grind this is to craft many unique foods in a row, then put the unique foods in your inventory, go near a passive mob, spam hit the passive mob with the healing axe, and eat your unique foods.

**Comment by bedrill64** [comment]
I'm also against nerfing the healing axe, the food finding and crafting is main challenge of stone-steam age, with cooking for blockheads it is going to become even easier, so people actually might not even want to grind the axe quest, just because food crafting gonna be convenient. This is not rpg survival pack for the food to be problem for the entire playthrough, there are better things to do later in game than worrying about burger crafting or creating right and left twix factory in your basement.

**Comment by Prometheus0000** [comment]
Remove tin cans

**Comment by dimondmine2** [comment]
Because cooking for blockheads may arguably make completing the healing axe easier, making the healing axe quest harder may be something to discuss AFTER cooking for blockheads has been used, tested, and familiarized by players so we can get a more informed discussion on making the quest harder (if the consensus is that cooking for blockheads has made it too easy).

**Comment by leagris** [comment]
Players pursuing the Healing axe are doing so mostly because they don't enjoy dealing with food as a gameplay.

Do you **really** think that these player will enjoy more nerves on the healing axe quest or just screw it and just cheat one at the beginning of the game instead and entirely bypass your nerved recipes and quest?

If someone does not like broccoli, do you serve more broccoli or politely propose alternatives?

**Comment by Prometheus0000** [comment]
I also did the quest for completion's sake.

**Comment by leagris** [comment]
> I also did the quest for completion's sake.

I like broccoli, and completed the quest ASAP as well.

But as a housewife, I prefer not to deal with those households burdens of ~~cooking~~, ~~cleaning~~ or ~~laundry~~ in a game.

**Comment by basdxz** [comment]
I see this as a lack of polish on the part of foodstuffs. A large chunk of the playerbase doesn't care about eating and just want to get on with the factory building and progression. Which is fair, and the healing axe is a great object with respect to players time.

But there are countless mechanics that players see as "Broccoli", I'd argue that this pack is more Broccoli than not. To some extent, the tedium and complexity is what brought people to GT:NH in the first place.

The healing axe and it's quest is fine as it is, considering if you wish to survive a pair of lunch bags packed with random morsels is more than enough to last a player till either the MV when the healing axe is cratable or later when canned food becomes a suitable alternative.

Even removing the healing axe altogether will not make the players be incentivized to craft better foods, fewer food for extra hunger is not valued as much as just filling the bar with the shortest number of steps.

TL;DR
Make good foods worth eating. Dezils marshmallow is an extreme example of this, let foods give you substantial speed, haste or defence buffs to promote them.

**Comment by TimeConqueror** [comment]
Healing axe is a good example of such kind of things, because of which you make a progression through the pack. Useful items that make your life much easier, if you work hard enough to achieve them. Working hard for more hard work is not smth, that should be an incentive. So these items are just a good reward for your hard work on some stage of progression.

**Comment by Methes** [comment]
I'd say the food sources are in a good spot atm.
To get the Axe you have to explore a lot and setup basic food infrastructure.
Axe is great for running and general activity around your base.
Regular food is great for mining - you don't want to wait 15 seconds for your hunger to recharge from Axe
Tin Cans are great for everything but inventory management is more tedious.

**Comment by Dream-Master** [comment]
yea the healing axe quest will not removed.

**Comment by interestingusernam3** [comment]
>I see this as a lack of polish on the part of foodstuffs. 

Solution: add pierogi

**Comment by repo-alt** [comment]
![Uploading image.png…]()
